test/fixtures/api/resource-parameters.json

```diff
@@ -84,44 +84,37 @@
                         "value": {
                           "element": "enum",
                           "attributes": {
-                            "samples": {
+                            "default": {
                               "element": "array",
```

Comment:
The default value here is still being wrapped in an array which will need to be removed.